tasks_8/mini_bash_5.py

```diff
@@ -0,0 +1,153 @@
+"""
+Задача 5*
+Реализовать программу, позволяющую осуществлять управление файлами:
+копирование, создание, удаление, переименование.
+Необходимо предусмотреть возможность смен директории.
+Изначально используется текущий каталог запуска скрипта программы.
+"""
+
+import os
+import sys
+import shutil
+
+
+def show_help(*args):
+    print("""
+    help  - list of commands
+    pwd   - present working directory
+    ls    - list directory content
+    mkdir - make directory
+    touch - make file
+    cd    - change directory
+    cp    - copy file or directory
+    rm    - remove file or directory
+    mv    - move file or directory
+    exit
+    """)
+
+
+def present_working_directory(*args):
+    path = os.getcwd()
+    print(path)
+
+
+def list_directory_content(*args):
+    content = os.listdir(path=".")
+    for file in content:
```

Comment:
What about to dig deeper? 
For what need *arg and what if I need `ls` for not a current directory?